meson.build

```diff
@@ -465,6 +413,46 @@ symb_defs = {
     'cblas_?dotc_sub':  {'def': ['CBLAS', 'FORCE_USE_STACK', 'CONJ']},
 }
 
+# config.h file generation
+
+run_command('c_check', 'Makefile.conf', './build/_config_1.h', 'gcc')
+run_command('f_check', 'Makefile.conf', './build/_config_1.h', 'gfortran')
+
+getarch = executable('getarch', ['getarch.c', 'cpuid.S'])
+
+_config_2h = custom_target('_config_2h',
+                           output: '_config_2.h',
+                           command: [getarch, '1'],
+                           depends: [getarch],
+                           capture: true,
+                          )
+
+_config_for_getarch_2nd_h = custom_target('_config_for_getarch_2nd_h',
+                                          output: '_config_for_getarch_2nd.h',
+                                          command: [find_program('cat'), './_config_1.h', _config_2h],
+                                          depends: [_config_2h],
+                                          capture: true,
+                                         )
+
+getarch_2nd = executable('getarch_2nd',
+                         ['getarch_2nd.c', _config_for_getarch_2nd_h],
+                         c_args: ['-DGEMM_MULTITHREAD_THRESHOLD=4', '-DBUILD_WITH_MESON']
+                        )
+
+_config_3h = custom_target('_config_3h',
+                           output: '_config_3.h',
+                           command: [getarch_2nd, '1'],
+                           depends: [getarch_2nd],
+                           capture: true,
+                          )
+
+config_h = custom_target('config_h',
+                         output: 'config.h',
+                         command: [find_program('cat'), _config_for_getarch_2nd_h, _config_3h],
```

Comment:
Minor nit, better to use a `python` script than `cat` for cross compatibility

Reply:
Done! Moved to a Python script.